### ISSUE TITLE
[Question] What is the difference between this and Reactive Resume?

### ISSUE THREAD

**Issue by Cjkeenan** [SHENG-X/simple-resume#32]
I saw that this was initially fork of [Reactive Resume](https://github.com/AmruthPillai/Reactive-Resume) so I was wondering what are the main differences between the two apps? I see that their webpage layouts' are slightly different but other than that I cannot find a reason to choose one over the other. Thank you for any help you can provide.

**Comment by SHENG-X** [comment]
Hello @Cjkeenan,  thanks for your comment. The main differences between these two apps are the simple resume app is focused on building a professional Canadian resume, other than that it also intended to provide a better UX for technical and non-technical users. There are more changes than just the layout, for example, the markdown Textarea in the Reactive Resume app is fully replaced with the more friendly editable text for non-technical users. Animations are added for moving and delete an item. There are more internal data structure changes too. Other than that, I'll be super active in maintaining this project.

Hopefully, that answers your question.

**Comment by Cjkeenan** [comment]
yeah that's exactly what I was looking for. Great work by the way, the only issues I have had thus far was having multiple levels of bullets, which I just manually got around with the use of spaces and hyphens, and being able to move the sections around at will, which maybe could be done with the use of a custom template. Otherwise I am very happy.

**Comment by SHENG-X** [comment]
@Cjkeenan Thanks for your feedback. I'll add them to the new feature list, and I'll start work on them as soon as I can.

**Comment by Cjkeenan** [comment]
also is there a spellchecker in use?

**Comment by Cjkeenan** [comment]
> Hello @Cjkeenan, thanks for your comment. The main differences between these two apps are the simple resume app is focused on building a professional Canadian resume

What exactly is a "Canadian resume"? I've never heard of that term.

**Comment by SHENG-X** [comment]
@Cjkeenan Currently, there is no spellchecker implemented yet. I was thinking about implementing the spellchecker, but I did not get it to work. It probably will take a bit of time for me to get there. 

I am sorry for the confusion on the "Canadian resume".  My definition for "Canadian resume" is a resume that works for people in Canada since I am from Canada.

**Comment by SHENG-X** [comment]
Hello @Cjkeenan, the multilevel list and spellchecker features are fully supported in the lastest app.

**Comment by Cjkeenan** [comment]
@SHENG-X Is it live on the website, and if so how do I use the multilevel list because tabbing doesn't seem to be working.

**Comment by SHENG-X** [comment]
@Cjkeenan Yes, it is live on the website. https://simpleresume.ca/. Tab should work. The app is serviced from cache first so you probably do not have the latest code on your browser. To solve the problem for now,  can you try hard reload the page or open the site in incognito mode? 

You should see something like this.
<img width="327" alt="Screen Shot 2020-07-01 at 6 40 52 PM" src="https://user-images.githubusercontent.com/23110925/86306999-80634a00-bbca-11ea-9183-7426a45b8d03.png">

**Comment by Cjkeenan** [comment]
OK so I reset the data for the site and its now working but when I import my resume, all of the formatting seems to be screwed up, see image below:
![image](https://user-images.githubusercontent.com/10240016/86308300-a9d1a500-bbcd-11ea-8995-4ccc46ae8648.png)
Any idea what is causing this? I exported right before I cleared the site data.

**Comment by SHENG-X** [comment]
Hello @Cjkeenan, you see this because the internal data structure changed completely.  At this point, there is no backward compatibility so you will need to copy past or re-enter your descriptions. I am sorry for the inconvenience :(.

**Comment by Cjkeenan** [comment]
No problem, just wondering.

**Comment by SHENG-X** [comment]
let me know if you have other feature requests or some templates that you would like to use. I'll try my best to get them implemented :).

**Comment by Cjkeenan** [comment]
Yeah my biggest feature request would probably be a custom template that you could format like a table, specify number of rows and how many columns in each row and being able to resize those columns. Then being able place the categories into those grid areas. Also with resizing the grid, having some sort of snapping would be nice, either snapping to 1/2, 1/4, 1/3, etc of page and also snapping to other elements on the page. I hope that makes sense, if not , please ask for clarification.

**Comment by SHENG-X** [comment]
I think that I get what you need. Is it something that works like Wix? If so, it is out of the scope of this application.

**Comment by Cjkeenan** [comment]
I am not sure as I have never used Wix. The reason I ask for this is because that is previously what I was doing in word with the use of tables with invisible borders to put data in certain sections.

Maybe a better way to put it is, is there any way to modify or make new Templates? I am relatively technical so if you could point me in the direction of how those templates are created, maybe I could try and modify them.

**Comment by SHENG-X** [comment]
In order to create a new template, you need to check out the branch first. After you check out the branch and open the code on your editor, you will see a template folder
<img width="242" alt="2020-07-03 at 6 32 PM" src="https://user-images.githubusercontent.com/23110925/86502774-bd127b00-bd5b-11ea-87d3-d8b62a0a43dd.png">. You can add a new template by following the structure of other templates. After you are happy with your template, you can check in the code if you want to share it with others.

**Comment by SHENG-X** [comment]
@Cjkeenan or you can send me a screenshot of what your template suppose to look like then I can make it for you.

**Comment by Cjkeenan** [comment]
> @Cjkeenan or you can send me a screenshot of what your template suppose to look like then I can make it for you.

I did not have a specific template in mind, just would like to optimize my two-column resume does not have big blocks of white space is all.

On another note, I just thought of a nice feature request. Potentially would it be possible to interface with cloud platforms like OneDrive or Google Drive as a source or destination of imports and exports respectively?

**Comment by SHENG-X** [comment]
I see. For now, the only way to reduce white space is to dive into the source code and modify the space when you need it. 

yeah, this could be a good feature to have. I'll do some investigation on it first.
